swift/Sources/PiranhaKit/CleanupStaleFlags/StaleFlagCleaner.swift

```diff
@@ -327,11 +249,14 @@ class XPFlagCleaner: SyntaxRewriter {
             }
         } else if let functionCallExprNode = FunctionCallExprSyntax.init(node) {
             // handles flag API calls
-            let api = flagApiType(of: functionCallExprNode)
+            guard let api = flagApiType(of: functionCallExprNode) else {
+                return cache(expression: Syntax(functionCallExprNode),
+                             with: Value.isBot)
+            }
             var value = Value.isBot
```

Comment:
This isBot is used only when api returns `.testing`?

Reply:
Yes, that understanding is correct.